features/directory_structure.feature

```diff
@@ -3,16 +3,18 @@ Feature: Directory Structure
   Specs are usually placed in a canonical directory structure that describes
   their purpose:
 
-  - Model specs reside in the `spec/models` directory
-  - Controller specs reside in the `spec/controllers` directory
-  - Request specs reside in the `spec/requests` directory. The directory
-    can also be named `integration` or `api`.
-  - Feature specs reside in the `spec/features` directory
-  - View specs reside in the `spec/views` directory
-  - Helper specs reside in the `spec/helpers` directory
-  - Mailer specs reside in the `spec/mailers` directory
-  - Routing specs reside in the `spec/routing` directory
-  - Job specs reside in the `spec/jobs` directory
+  - [Model specs](../model-specs) reside in the `spec/models` directory
```

Comment:
For relish these can all just be `(model-specs)` etc which I'd prefer as it's clearer it will work, could you change them and then I'm happy to merge.

Reply:
Sure, I took the syntax from here: https://github.com/rspec/rspec-rails/blob/e7b9103c4e7e878d1fe0bf8e0044e6b6f310ea5c/features/matchers/have_http_status_matcher.feature#L6-L7